### ISSUE TITLE
Qobuz integration

### ISSUE THREAD

**Issue by 9acca9** [benkaiser/stretto#197]
Well, i just want to ask if you think in add this posibility later.
Thanks.

**Comment by benkaiser** [comment]
Hey @9acca9 , not on my radar, especially since it isn't used here in Australia. Open to pull requests though. I'm guessing you're wanting some kind of importing functionality?

**Comment by 9acca9** [comment]
Yes, it will be nice if i can import the artist, album, and playlist followed. If it is possible.  I will open a pull request! Thanks!
